arbitrum-docs/for-devs/concepts/differences-between-arbitrum-ethereum/overview.mdx

```diff
@@ -0,0 +1,44 @@
+author: jose-franco
+sme: jose-franco
+target_audience: developers who want to build on Arbitrum
+content-type: concept
+---
+
+Arbitrum is designed to be as compatible and consistent with Ethereum as possible, from its high-level RPCs to its low-level bytecode and everything in between. Decentralized app (dApp) developers with experience building on Ethereum will likely find that little-to-no new specific knowledge is required to build on Arbitrum.
+
+This section describes the differences, perks, and gotchas that devs are advised to be aware of when working with Arbitrum. This first page serves as an overview of where you might these differences, with links to the relevant pages when needed.
+
+## Block numbers and time
+
+Time in L2s is tricky. The timing assumptions one is used to making about Ethereum blocks don't exactly carry over into the timing of Arbitrum blocks. See [Block numbers and time](/for-devs/concepts/differences-between-arbitrum-ethereum/block-numbers-and-time.mdx) for details about how block numbers and time are handled in Arbitrum.
+
+## RPC methods
+
+Although the majority of RPC methods follow the same behavior than in Ethereum, some methods might produce a different result, or add more information, when used on an Arbitrum chain. You can find more information about these differences in [RPC methods](/for-devs/concepts/differences-between-arbitrum-ethereum/rpc-methods.mdx).
+
+## Solidity support
+
+You can deploy Solidity contracts onto Arbitrum just like you do Ethereum. There are only a few minor differences in behavior. Find more information about it in [Solidity support](/for-devs/concepts/differences-between-arbitrum-ethereum/solidity-support.mdx).
+
+## Fees
+
+The fees an Arbitrum transaction pays essentially work identically to gas fees on Ethereum. Arbitrum transactions must also, however, pay a fee component to cover the cost of posting their calldata to the parent chain (for example, calldata on Arbitrum One, an L2, is posted to Ethereum, an L1). Find more information about the two components of gas fees in [Gas and fees](/arbos/gas.mdx) and [L1 pricing](/arbos/l1-pricing.mdx).
+
+## Cross-chain messaging
+
+Arbitrum chains support arbitrary message passing from a parent chain (for example, a Layer 1 (L1) like Ethereum) to a child chain (for example, a Layer 2 (L2) like Arbitrum One or Arbitrum Nova). These are commonly known as "L1 to L2 messages". Developers using this functionality should familiarize themselves with how they work. Find more information about it in [L1 to L2 messaging](/arbos/l1-to-l2-messaging.mdx).
+
+Similarly, Arbitrum chains can also send messages to the parent chain, although these follow more strict rules. Find more information about them in [L2 to L1 messaging and the outbox](/arbos/l2-to-l1-messaging.mdx).
```

Comment:
"although these follow more strict rules"
I'm actually not sure what this is referring to (L2 to L1 messaging is equally arbitrary)  — can we remove or am I missing something?

Reply:
I'm not sure what I was thinking about, but my brain might have bugged for a second there. I just removed it.